With the fix from the most recent [change](https://github.com/NixOS/nixpkgs/compare/3710842eb3b344e12a5dcb40f98c7c8ebe10ac3d..04b48b2b43318f4554145f7856758a44eedc67ff) the build starts for me too.  Unfortunately on aarch64-darwin the build fails when doing:

```
$ git clone git@github.com:NixOS/nixpkgs.git && cd nixpkgs
$ nix-shell -p nixpkgs-review --run "nixpkgs-review pr 263530"
...
       >   = note: ld: framework not found AppKit
       >           clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
...
```

<details><summary>Full error</summary>

```
$ nix build --nix-path nixpkgs=/Users/janis.koenig/.cache/nixpkgs-review/pr-263530-1/nixpkgs nixpkgs-overlays=/tmp/tmpt3lpj51z --extra-experimental-features nix-command no-url-literals --no-link --keep-going --no-allow-import-from-derivation -f /Users/janis.koenig/.cache/nixpkgs-review/pr-263530-1/build.nix
error: builder for '/nix/store/d128hqk8l31101wyqfrqljf2yp1vpynx-atuin-17.0.0.drv' failed with exit code 101;
       last 10 log lines:
       >    Compiling clap_complete v4.4.4
       >    Compiling runtime-format v0.1.3
       > error: linking with `/nix/store/sa6hywsm1mqfyd1xakyzv4ljjsb3hawh-clang-wrapper-11.1.0/bin/cc` failed: exit status: 1
       >   |
       >   = note: LC_ALL="C" PATH="/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/bin:/nix/store/4qdazhg6d6ck4ry9wh099f25hfgkpg5k-cargo-1.72.1/bin:/nix/store/yvljjfcbmk1grdjnplyyjymz642khbsm-cargo-auditable-0.6.1/bin:/nix/store/62p2hzr9s4hpimwfpzd9m6ca3zlyvhm0-auditable-cargo-1.72.1/bin:/nix/store/4qdazhg6d6ck4ry9wh099f25hfgkpg5k-cargo-1.72.1/bin:/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/bin:/nix/store/sa6hywsm1mqfyd1xakyzv4ljjsb3hawh-clang-wrapper-11.1.0/bin:/nix/store/ljz45vk739778lw2xaw2jvy7ihip2q07-clang-11.1.0/bin:/nix/store/943sxl4vcfpfg6xaagxvgwbgz9scl7lc-coreutils-9.3/bin:/nix/store/m488d5iwzn93bdk1j5gxl77k3zb8y285-cctools-binutils-darwin-wrapper-11.1.0-973.0.1/bin:/nix/store/x5bm27csww06snmzrsqhxfzl0i8mad5c-cctools-binutils-darwin-11.1.0-973.0.1/bin:/nix/store/943sxl4vcfpfg6xaagxvgwbgz9scl7lc-coreutils-9.3/bin:/nix/store/ngyiha629vs37hb2pwmk489ndf7nmh20-findutils-4.9.0/bin:/nix/store/d1gsih58dvmj2da2im0c5k3rz2x6zhff-diffutils-3.10/bin:/nix/store/fh1327f06qipzcnv2di22vssbbzk1hrz-gnused-4.9/bin:/nix/store/kxbn7dv0i7p0ylsbr53j7z0gf45kim78-gnugrep-3.11/bin:/nix/store/qllkm9swhw8fgs46v2mk64kl88cm7jh8-gawk-5.2.2/bin:/nix/store/fpmgc7y2dihni7kpwvgaq46fjcds6my1-gnutar-1.35/bin:/nix/store/5dyk8m2il19cm2q7bvrlvnd6957jpp6s-gzip-1.13/bin:/nix/store/722g4vgl3g935h805y7phifmdvq2xl8p-bzip2-1.0.8-bin/bin:/nix/store/5sbqambqp3w2sc5mm3b7ddgla5jp7v9a-gnumake-4.4.1/bin:/nix/store/zzpm4317hn2y29rm46krsasaww9wxb1k-bash-5.2-p15/bin:/nix/store/v2s7n510v8zf67ngj4962zjia6h25r88-patch-2.7.6/bin:/nix/store/w2hc8y9m36d2nwrvrf27yrvzpwc8pbcm-xz-5.4.4-bin/bin:/nix/store/ps2jbdzj9kxch1b77apis0i06chk91qw-file-5.45/bin" VSLANG="1033" ZERO_AR_DATE="1" "/nix/store/sa6hywsm1mqfyd1xakyzv4ljjsb3hawh-clang-wrapper-11.1.0/bin/cc" "-arch" "arm64" "/private/tmp/nix-build-atuin-17.0.0.drv-0/rustcsPrKoX/symbols.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.00.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.01.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.02.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.03.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.04.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.05.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.06.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.07.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.08.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.09.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.10.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.11.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.12.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.13.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.14.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.atuin.7e362561814cb31e-cgu.15.rcgu.o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b.204usguurlruy5wj.rcgu.o" "-L" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps" "-L" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/release/deps" "-L" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/build/ring-607f95be38603140/out" "-L" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/build/libsqlite3-sys-afadc61900505297/out" "-L" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/build/ring-a2af7b1c918e3f18/out" "-L" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcli_clipboard-15d423ce774fe115.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libobjc_foundation-903fd0d6bb051787.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libblock-f15a63b62d043369.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libobjc_id-9cf4c6198eef0785.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libobjc-7b5691a0c2b24591.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libmalloc_buf-d02a7bcccee563b5.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libbip39-6cf19bceb9a418f9.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libpbkdf2-1dd4c284f6056fbc.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librustc_hash-0811f27b46590085.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libatuin_server-e2cecff77a79854f.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtower_http-dbb850276b2c6255.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libaxum-427fd4317b8eb428.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libserde_path_to_error-5534d5e769f5c6c6.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libmatchit-04e9bdb4291b5af7.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsync_wrapper-9cf4b8c3bfc4d5ad.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libaxum_core-eb535c53e44ad4d3.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtower-fc2b11b81c52b2d8.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libpin_project-ae71bbe60c180154.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtower_layer-18d7fb12b60aed95.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtracing_subscriber-328bc610598e5499.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnu_ansi_term-c7a2a4b14a1af1df.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liboverload-ff6c0c067e3ec7dd.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsharded_slab-92688526610e397b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libmatchers-a240f2830626c8d8.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libregex_automata-ed06e7d3b8dcab85.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libregex_syntax-e9bf34a8e7ddc86d.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libatuin_server_postgres-66a4009187e001bf.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libatuin_server_database-05c51e1ae620a287.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libratatui-579fb4dadad9e41d.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcassowary-17041b33072f0182.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libunicode_segmentation-7b17c40f8c2df9a8.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcrossterm-50cb46053da92738.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfiledescriptor-81509ce32f7c6336.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsignal_hook-832fdc0c387fef16.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libstrum-766bf84dfa83fc13.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfuzzy_matcher-97efe702ff01f30a.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libthread_local-2803414042387bf3.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libindicatif-f9d07b3ddd2cd2f1.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libportable_atomic-d9dbace19ef0dca1.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnumber_prefix-026b83e318111665.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libconsole-419bb669652e6b00.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libunicode_width-5571cc2234fde35a.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libruntime_format-eec98568e7eb8391.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librpassword-fc2c6e357d5517bd.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librtoolbox-669828462f3952ab.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcolored-a7ed88ec70935dbd.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libenv_logger-b63e19a56c0632dc.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtermcolor-ccd14b21ad7c817c.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libis_terminal-be3e7fa0207d96fd.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librustix-27346ce5db049cdc.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liberrno-64d12d8770fd1a24.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhumantime-789f3700b7217bdb.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libatuin_client-a38a671e85f21d0b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libshellexpand-09de8e57cbc02ad0.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libdirs-5c1eb42cedad006a.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libdirs_sys-09cf18d6b32ce735.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liboption_ext-7cb5f4b9806b6fcc.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libserde_regex-44c7088966b1a6b7.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libinterim-4968700b6ed92cc5.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liblogos-2fc5f03fd3f72bba.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liburlencoding-26a9a913e10adb44.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libparse_duration-6c5b4c685251a624.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnum-d742cfdb253b944d.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnum_rational-8396c1ec6aacd0d2.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnum_iter-2715d756a6ae8d08.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnum_complex-5506dc1c2d912e32.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnum_bigint-a31b08429faf63c1.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnum_integer-09db673498b23392.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libconfig-ed7e50f1972f596a.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libpathdiff-e09a0cc27e7923a6.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtoml-4b8bac5313181e58.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures-4cb5f62661865f8d.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librusty_paserk-6e5b6dd6544695fe.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libargon2-925c55d8338cc09b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libpassword_hash-70c348c5d3aed96b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libchacha20-63e5c1cb1a06ede2.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libed25519_dalek-fc14ad0e83896530.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcurve25519_dalek-09e072ba57518301.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libed25519-519cbb75202e415c.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsignature-a4b45737678bb364.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libblake2-0ed7a7f335e793eb.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librusty_paseto-0fe10feed1e707c9.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libed25519_dalek-14570db05dd1d5d6.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsha2-a37b36ee83d90a47.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libblock_buffer-784f419d38d724b3.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librand-ddefc27b92167a19.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librand_chacha-c4cfcf220a2b2baa.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcurve25519_dalek-c74463cc432f13ae.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librand_core-e526919ae40e0a83.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libgetrandom-0e7a0dfda2348cb1.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libed25519-fe6ac005da9fc685.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsignature-ceab90e4869b0334.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libchacha20-6b41323122da8899.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcipher-93a732612ee54262.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libblake2-9610f9026890b61c.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcrypto_mac-246a1babd9859178.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libdigest-256cb6c59eb2cf9a.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libbase64-c59ff236dfb1a2a6.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libring-ff027ef2f8f1cdf5.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libuntrusted-c595a34718f45b92.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libbase64ct-71b721b228d722f3.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libminspan-65baae5d2f08a6e6.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libdirectories-7b2ce73def2a0c2d.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libdirs_sys-698f7b914bf2d962.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librmp-4d7503ba1c7ae310.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsql_builder-fdc70e981f28baba.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libanyhow-7868243d0bf36984.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libregex-94cf068096effe91.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libregex_automata-45a4945bc92d9dc1.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libaho_corasick-36d6b9fea9d1b801.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libregex_syntax-c97bb5f5c14b812e.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liblazy_static-68523cf2dcda5679.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfs_err-9baff1e000c5134b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcrypto_secretbox-f4f63aeae0811c71.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsalsa20-f5363848e5122d03.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libpoly1305-345aebef2f247a98.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libopaque_debug-9395a7be3aa3981f.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libuniversal_hash-3100289c68b3bdd3.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcipher-bf71e07f0f733c38.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libinout-7158682e11c89b65.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libaead-2c80ba739fe9bf05.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsemver-5419c71f00547ccc.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libatuin_common-599c5d4d1262a00b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsqlx-3e282c603eb07e98.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsqlx_sqlite-d283b62774aa64ef.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_executor-c2f153eddd687a4a.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libflume-1de3ec006cef901b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libspin-260f25469a8acaa1.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_intrusive-0766f41311982e1b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liblibsqlite3_sys-a1d037907914d67f.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsqlx_postgres-8e486d552a86530c.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libdotenvy-dc2f458a9deb259e.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcrc-2a85f49b88e2da1c.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcrc_catalog-7610df28d3eeaf51.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhex-b652076504690dcd.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libwhoami-7cb8d92b0e2f5dab.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhome-44c7752cc76b747c.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libbitflags-572ffab22c6a48b2.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libmd5-e5f2cfc885b1c9dd.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libbyteorder-015f4954a3a37b3e.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libatoi-48db5b795b8a0cfd.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnum_traits-b331e009cf15afe5.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libstringprep-b23f8e4a59ad0007.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfinl_unicode-5882947f4cd11895.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhkdf-137844528976523b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhmac-ad1bedcce8eaf0b2.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsqlx_core-3b235e2ad4fe47d5.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libwebpki_roots-2f868056aae5929e.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libthiserror-d53acd8ce2316777.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libindexmap-98c6d5e0bf99a79c.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libequivalent-7171fffad0269b9b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsha2-fc451b6723f69df1.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libdigest-00574cf82dbe8d65.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsubtle-3b1d876a0b63b39e.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libblock_buffer-c08e22273b0b9619.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcrypto_common-7a82e44c3fe955ff.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libgeneric_array-0675dadaa7283375.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtypenum-b4b89cd15d2868e1.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libzeroize-46e31383968fd54d.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsqlformat-60c3c83603d0551d.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libunicode_categories-fe995629a313f96e.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnom-4ac7dced5cec1a05.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libitertools-bed0ebc60bcc05bc.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhashlink-a72c85a8c49a074e.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhashbrown-069bf6323efb1538.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libahash-6ca1adfe656c9941.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libzerocopy-8fd2e41a0c86b34c.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liballocator_api2-8fb4e7087ec6e3ca.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcrossbeam_queue-138bcfb5064ffa65.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcrossbeam_utils-13b1553355dcc592.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libeither-fa23cdc576b67db6.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libevent_listener-f6b49dd125f5d5e0.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librand-67ae5394c555f7d3.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librand_chacha-c32eceeef7a4a5d3.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libppv_lite86-9004b99c19fed72e.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librand_core-2e8ff12c1ec172c5.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libuuid-a60b83985766c1f0.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtyped_builder-5a52dc3517bd2292.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtime-0f9d44d413e5f925.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnum_threads-8084435ffde6d5b8.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtime_core-0d99ae96551bbe30.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libderanged-9baa08a1c9ec65a6.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libreqwest-c6a6bb1b9cbf1d86.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhyper_rustls-bacbdad775c29313.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libserde_urlencoded-c83cd0f3cb9a83f0.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librustls_native_certs-b9bbb7bf7eaef868.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librustls_pemfile-1840cef02cf3b9ce.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsecurity_framework-2abb8673867965f5.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsecurity_framework_sys-6cd9995c36d7f530.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libbase64-c0111b5fbaf914a2.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsystem_configuration-cd33dcd09c117ae1.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libbitflags-6dbdffdc03798ec1.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsystem_configuration_sys-400b93d67fed1a52.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcore_foundation-79e13779d3efd3b8.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcore_foundation_sys-27a5e4aa855f8177.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libipnet-b2ed360362d01e58.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtokio_rustls-1c422320f0b80780.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/librustls-6d0a2355573406bc.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsct-5e7673b16b5cba5f.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libwebpki-2adb8b55f593dcff.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libring-d50b6952f9c6dd19.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libgetrandom-043ac9fb0a7d86ec.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libuntrusted-f0badc7278b1b255.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libmime-2aca27b9ef85f45f.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libencoding_rs-479ede82e25bfd42.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libserde_json-ae4c57dc5c60d811.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libryu-24aaf65e85283f35.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libserde-d514f7dd3d224cc6.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhyper-92376019a9eb4e44.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libwant-b0dd9acab138e9c4.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtry_lock-f3c03d5f955eba4d.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsocket2-cc2a26f4a4b1a146.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhttparse-f21276899e6ccf54.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libh2-9d79cdd4ec666167.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libindexmap-8cfbd2634a5094e8.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhashbrown-4172533b45466665.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtokio_util-716357e82e5d8861.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtower_service-f5f5decd3220f138.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtracing-e86322afe2460405.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtracing_core-f702d98d29a30e71.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhttpdate-aacd310ecc9bed16.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_util-3fafb894110dc6c6.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libmemchr-a928dbc44bd314f3.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_io-e50371587ae3e05d.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libslab-9aa967ac90832106.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_channel-92dfa1cfef1cd4b4.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_sink-393af22bf15af974.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_task-e465552c4933339b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libpin_utils-bb1f7129a0f68291.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtokio-84d10365ff1a97ba.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsignal_hook_registry-731c7af75f69f5c8.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libnum_cpus-d8a5940b0617c10d.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsocket2-d15b6b719de0d203.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libmio-ed81880d2911a665.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libparking_lot-6fa2e132a641417f.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libparking_lot_core-37ff8c391df41c07.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liblibc-f9e69c1559976fd5.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcfg_if-3ae4d3f00b8ddb99.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libsmallvec-f3d29d499cdc4389.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liblock_api-292c0566095a73b5.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libscopeguard-217505abb9d07a1c.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhttp_body-646de07b57f5c7bd.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libpin_project_lite-7d11b4c51a5c6d1e.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfutures_core-dd72be2617765876.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liburl-c7171adb04681f5e.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libidna-650745c342ee28d9.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libunicode_normalization-3aa718956481a895.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtinyvec-d0b4a360c2eeb5f7.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libtinyvec_macros-2bccf4f4b6b926e8.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libunicode_bidi-3058543e1ee102e9.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libform_urlencoded-136ddc53cf4d1f89.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libpercent_encoding-08dc949b7121d89b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libhttp-a31a420825775020.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libitoa-cef4681f67f31e13.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libbytes-ad3d6394b4c0a237.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libfnv-9f0f726b3cb4034d.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/liblog-b70425f0c0fa4c87.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libclap_complete-3f050b62274d9c23.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libeyre-fc559536f7efd738.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libindenter-4d72019f36b61c96.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libonce_cell-d4336360b86af051.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libclap-893b2be099fb162e.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libclap_builder-437a092463f1087b.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libstrsim-60ebafad46b5bb22.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libanstream-5caee9a7f6e0eccd.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libanstyle_query-c91e2d63835b9144.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libcolorchoice-929202d9278563f1.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libanstyle_parse-aa69600cc889ca9c.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libutf8parse-9c66f72d864c6806.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libclap_lex-297eb8b279deabcc.rlib" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/libanstyle-c37110054989e86f.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libstd-b7feb6b9c58672f5.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libpanic_unwind-c27da09bad3b3fae.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libobject-5240b66afa38f56d.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libmemchr-e21fc1725afcb42d.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libaddr2line-9ebab353dfc1fbab.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libgimli-3213edd3033fc2c2.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/librustc_demangle-1bc48ce882a6a616.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libstd_detect-2efd6ba151d15c9c.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libhashbrown-4fa615fdfc9b841f.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/librustc_std_workspace_alloc-29a03b7641b70a59.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libminiz_oxide-25f7672ef74e63b8.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libadler-5316a8c506419d93.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libunwind-ba51cbe539af61d2.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libcfg_if-b5f30008661136ff.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/liblibc-8fa239b056c79067.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/liballoc-56ba81188f3326c3.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/librustc_std_workspace_core-c5c917b00a5a4741.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libcore-26ab7e88c331ffa5.rlib" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib/libcompiler_builtins-1f437443701d86bd.rlib" "-framework" "AppKit" "-framework" "Foundation" "-lSystem" "-lobjc" "-framework" "Security" "-framework" "CoreFoundation" "-framework" "SystemConfiguration" "-framework" "Security" "-framework" "SystemConfiguration" "-framework" "CoreFoundation" "-liconv" "-lSystem" "-lc" "-lm" "-L" "/nix/store/8zq6sng72fx8hpnmcxwfgvr9vx4r1j5f-rustc-1.72.1/lib/rustlib/aarch64-apple-darwin/lib" "-o" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin-6e35fc75033dbb1b" "-Wl,-dead_strip" "-nodefaultlibs" "/private/tmp/nix-build-atuin-17.0.0.drv-0/source/target/aarch64-apple-darwin/release/deps/atuin_audit_data.o" "-Wl,-u,_AUDITABLE_VERSION_INFO"
       >   = note: ld: framework not found AppKit
       >           clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
       >           
       >
       > error: could not compile `atuin` (bin "atuin") due to previous error
       For full logs, run 'nix-store -l /nix/store/d128hqk8l31101wyqfrqljf2yp1vpynx-atuin-17.0.0.drv'.
error: 1 dependencies of derivation '/nix/store/whcvkljv81w25ymykn0qwv7670x4fbzh-review-shell.drv' failed to build
```

</details>

I tried adding `AppKit` to the dependencies and optional `buildInputs` for Darwin similar to 40f68f68f7b010024772a84fdaa7828267d78ea6.  This fails quite curiously (to me, a Nix noob) with:

```
Function called without required argument "AppKit"
```

<details><summary>Full log</summary>

```
$ nix-env --extra-experimental-features no-url-literals --option system aarch64-darwin -f <nixpkgs> --nix-path nixpkgs=/Users/janis.koenig/.cache/nixpkgs-review/rev-04b48b2b43318f4554145f7856758a44eedc67ff-dirty/nixpkgs nixpkgs-overlays=/tmp/tmpg4j9lwab -qaP --xml --out-path --show-trace --no-allow-import-from-derivation --meta
error:
       … from call site

         at /Users/janis.koenig/.cache/nixpkgs-review/rev-04b48b2b43318f4554145f7856758a44eedc67ff-dirty/nixpkgs/pkgs/top-level/all-packages.nix:287:11:

          286|
          287|   atuin = callPackage ../tools/misc/atuin {
             |           ^
          288|     inherit (darwin.apple_sdk.frameworks) Security SystemConfiguration;

       … while calling 'callPackageWith'

         at /Users/janis.koenig/.cache/nixpkgs-review/rev-04b48b2b43318f4554145f7856758a44eedc67ff-dirty/nixpkgs/lib/customisation.nix:139:35:

          138|   */
          139|   callPackageWith = autoArgs: fn: args:
             |                                   ^
          140|     let

       error: evaluation aborted with the following error message: 'Function called without required argument "AppKit" at /Users/janis.koenig/.cache/nixpkgs-review/rev-04b48b2b43318f4554145f7856758a44eedc67ff-dirty/nixpkgs/pkgs/tools/misc/atuin/default.nix:9'
$ git worktree remove -f /Users/janis.koenig/.cache/nixpkgs-review/rev-04b48b2b43318f4554145f7856758a44eedc67ff-dirty/nixpkgs
Traceback (most recent call last):
  File "/nix/store/d2x42bsvb0j842dihnfwqmc2ipaliwjp-nixpkgs-review-2.10.3/bin/.nixpkgs-review-wrapped", line 9, in <module>
    sys.exit(main())
             ^^^^^^
  File "/nix/store/d2x42bsvb0j842dihnfwqmc2ipaliwjp-nixpkgs-review-2.10.3/lib/python3.11/site-packages/nixpkgs_review/__init__.py", line 10, in main
    cli.main(command, args)
  File "/nix/store/d2x42bsvb0j842dihnfwqmc2ipaliwjp-nixpkgs-review-2.10.3/lib/python3.11/site-packages/nixpkgs_review/cli/__init__.py", line 334, in main
    return cast(str, args.func(args))
                     ^^^^^^^^^^^^^^^
  File "/nix/store/d2x42bsvb0j842dihnfwqmc2ipaliwjp-nixpkgs-review-2.10.3/lib/python3.11/site-packages/nixpkgs_review/cli/wip.py", line 13, in wip_command
    return review_local_revision(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/d2x42bsvb0j842dihnfwqmc2ipaliwjp-nixpkgs-review-2.10.3/lib/python3.11/site-packages/nixpkgs_review/review.py", line 582, in review_local_revision
    review.review_commit(builddir.path, args.branch, commit, staged, print_result)
  File "/nix/store/d2x42bsvb0j842dihnfwqmc2ipaliwjp-nixpkgs-review-2.10.3/lib/python3.11/site-packages/nixpkgs_review/review.py", line 310, in review_commit
    self.build_commit(branch_rev, reviewed_commit, staged),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/d2x42bsvb0j842dihnfwqmc2ipaliwjp-nixpkgs-review-2.10.3/lib/python3.11/site-packages/nixpkgs_review/review.py", line 172, in build_commit
    merged_packages = list_packages(
                      ^^^^^^^^^^^^^^
  File "/nix/store/d2x42bsvb0j842dihnfwqmc2ipaliwjp-nixpkgs-review-2.10.3/lib/python3.11/site-packages/nixpkgs_review/review.py", line 398, in list_packages
    raise NixpkgsReviewError(
nixpkgs_review.errors.NixpkgsReviewError: Failed to list packages: nix-env failed with exit code 1
```

</details>

--- a/pkgs/tools/misc/atuin/default.nix
+++ b/pkgs/tools/misc/atuin/default.nix
@@ -4,6 +4,7 @@
 , installShellFiles
 , rustPlatform
 , libiconv
+, AppKit
 , Security
 , SystemConfiguration
 , nixosTests
@@ -11,24 +12,24 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "atuin";
-  version = "16.0.0";
+  version = "17.0.0";
 
   src = fetchFromGitHub {
     owner = "atuinsh";
     repo = "atuin";
     rev = "v${version}";
-    hash = "sha256-Kh6aaWYV+ZG7Asvw5JdGsV+nxD+xvvQab5wLIedcQcQ=";
+    hash = "sha256-eGukHoh1S2CtehapSRUM3+cnVvTpslwvDCSXlrnC4+4=";
   };
 
   # TODO: unify this to one hash because updater do not support this
   cargoHash =
     if stdenv.isLinux
-    then "sha256-Ami88ScGj58jCCat4MMDvjZtV5WglmrlggpQfo+LPjs="
-    else "sha256-HQMZ9w1C6go16XGrPNniQZliIQ/5yAp2w/uUwAOQTM0=";
+    then "sha256-KlfNOZ5xDID+x0GohbLv+SJlPQRK6C5YzbbFnKvm2ng="
+    else "sha256-8OarzrMc7EiHEDDbs2XiggivebQ68yufnmhXsf6MBwM=";
 
   nativeBuildInputs = [ installShellFiles ];
 
-  buildInputs = lib.optionals stdenv.isDarwin [ libiconv Security SystemConfiguration ];
+  buildInputs = lib.optionals stdenv.isDarwin [ libiconv AppKit Security SystemConfiguration ];
 
   postInstall = ''
     installShellCompletion --cmd atuin \
@@ -44,6 +45,8 @@ rustPlatform.buildRustPackage rec {
   checkFlags = [
     # tries to make a network access
     "--skip=registration"
+    # No such file or directory (os error 2)
+    "--skip=sync"
   ];
 
   meta = with lib; {

--- a/pkgs/tools/misc/atuin/default.nix
+++ b/pkgs/tools/misc/atuin/default.nix
@@ -12,20 +12,20 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "atuin";
-  version = "17.0.0";
+  version = "17.0.1";
 
   src = fetchFromGitHub {
     owner = "atuinsh";
     repo = "atuin";
     rev = "v${version}";
-    hash = "sha256-eGukHoh1S2CtehapSRUM3+cnVvTpslwvDCSXlrnC4+4=";
+    hash = "sha256-HJRlZwvBra2D7TzVKvMWJ0Hf17QgIEcBDQEHhxdVLIM=";
   };
 
   # TODO: unify this to one hash because updater do not support this
   cargoHash =
     if stdenv.isLinux
-    then "sha256-KlfNOZ5xDID+x0GohbLv+SJlPQRK6C5YzbbFnKvm2ng="
-    else "sha256-8OarzrMc7EiHEDDbs2XiggivebQ68yufnmhXsf6MBwM=";
+    then "sha256-AhoXmEjXsi/OgFX3htOA6A/lWegUFlsywdotX3PDwcs="
+    else "sha256-/nCnZ64pM8oWVX9a4JCeCZRyuo7aVc8YaBVEMbiRsqE=";
 
   nativeBuildInputs = [ installShellFiles ];
 

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -285,7 +285,7 @@ with pkgs;
   _0x =  callPackage ../tools/misc/0x { };
 
   atuin = callPackage ../tools/misc/atuin {
-    inherit (darwin.apple_sdk.frameworks) Security SystemConfiguration;
+    inherit (darwin.apple_sdk.frameworks) AppKit Security SystemConfiguration;
   };
 
   automatic-timezoned = callPackage ../tools/system/automatic-timezoned { };

